src/Asn1Net/Asn1Net.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup Label="Build">
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Package">
+    <VersionPrefix>1.6.0</VersionPrefix>
+    <Authors>Peter Polacko</Authors>
+    <Product>Asn1Net</Product>
+    <Description>Managed library for reading and writing ASN.1 objects.</Description>
```

Comment:
Change to "Asn1Net is portable library written in C# that enables convenient yet powerful ASN.1 processing in .NET environment"